### PR TITLE
Combining the overview pages into tabs.

### PR DESCRIPTION
**Pull request by nakul-bhagat**
## Description

 - Add a simple description of the changes

## Jira Ticket

 - Ref a Jira Ticket if it exists.

## Before Merging!

 - [x] Check [staging environment](https://developer-v2.pd-staging.com/docs) to ensure changes look as intended.
 - [x] Ensure there is a review from DevFoundations and from Community.

### CODE REVIEW

File: docs/webhooks/01-Overview.md [hunk 1]
```diff
@@ -55,7 +65,7 @@ V3 webhooks are configured by creating a _webhook subscription_ which contains t
   }
 }
 ```
```
Comment (Jimbo1167) [inline docs/webhooks/01-Overview.md:67]: try `\` to force newline?

Comment (Jimbo1167) [reply docs/webhooks/01-Overview.md:67]: or &nbsp; after this line

Comment (nakul-bhagat) [reply docs/webhooks/01-Overview.md:67]: '&nbsp;' worked but not '\'

**Comment by Jimbo1167** [comment]
Just need a corresponding change in our private developer-site repo to redirect properly, otherwise, this is looking good!

File: docs/webhooks/07-Webhooks-Beta.md [hunk 2]
```diff
@@ -1,12 +0,0 @@
----
```
Comment (ChezCrawford) [inline docs/webhooks/07-Webhooks-Beta.md:1]: I kind of lost how to get to this page when we moved over to Stoplight Docs but there is still a need for a hidden "Beta" page where we can document new webhook events / payloads that have not yet been fully released.

File: docs/webhooks/09-V2-Overview.md [hunk 3]
```diff
@@ -2,14 +2,19 @@
 tags: [webhooks]
 ---
 
-
 # v2 Overview
 
+<!-- theme: warning -->
+> ### Migrate to V3 Webhook Subscriptions
+> If you are currently using V1/V2 webhook extensions and need to migrate them to [V3 webhook](../webhooks/01-Overview.md)  subscriptions, please follow our migration guide.
```
Comment (ChezCrawford) [inline docs/webhooks/09-V2-Overview.md:9]: nit: there is a double space here after the link

File: docs/webhooks/09-V2-Overview.md [hunk 4]
```diff
@@ -1147,6 +1136,7 @@ title: incident.assign
   ]
 }
 ```
+&nbsp;
+### Latest Version
 
-<!-- type: tab-end -->
-
+Please go to [V3 webhook](../webhooks/01-Overview.md)
```
Comment (ChezCrawford) [inline docs/webhooks/09-V2-Overview.md:1142]: nit: This working is weird. Maybe "Please see V3 webhooks"?

File: docs/webhooks/09-V2-Overview.md [hunk 5]
```diff
@@ -104,13 +109,10 @@ Type                     | Description
...
 More webhook types may be added in the future.
 
-### Examples
-
-<!--
-type: tab
-title: incident.trigger
--->
 
+### V2.Examples
```
Comment (ChezCrawford) [inline docs/webhooks/09-V2-Overview.md:113]: Is this supposed to be "V2.Examples" or "V2 Examples"?

File: docs/webhooks/10-V1-Overview.md [hunk 6]
```diff
@@ -5,15 +5,19 @@ tags: [webhooks]
...
 Webhooks let you receive HTTP callbacks when interesting events happen within your PagerDuty account. Details surrounding the interesting event will be sent via HTTP POST to a URL that you specify.
 
 PagerDuty currently supports incident-based webhooks. After adding a webhook URL to a PagerDuty service, the triggering of new incidents on that service will cause outgoing webhook messages to be sent to that URL. In addition, certain interesting changes to an incident's state will cause other types of incident webhook messages to be sent. Generally, any change to the `status` or `assignees` of an incident will cause an outgoing message to be sent.
 
-### Webhook Payload
 
-<!-- theme: info -->
-### Upgrade to v2 Webhooks
-> Looking for webhooks on notes, or more details in your webhook payload? Add a new __Generic V2 Webhook__ extension to receive the [v2 webhooks payload](../../docs/webhooks/06-v2-Overview.md).
+### Webhook Payload
```
Comment (ChezCrawford) [inline docs/webhooks/10-V1-Overview.md:20]: nit: on the V2 page you made this: "V2 Webhook Payload". Can we be consistent?

File: docs/webhooks/10-V1-Overview.md [hunk 7]
```diff
@@ -65,14 +69,10 @@ Type                     | Description
...
-
-### Examples
-
-<!--
-type: tab
-title: incident.trigger
--->
-
+&nbsp;
+### V1.Examples
```
Comment (ChezCrawford) [inline docs/webhooks/10-V1-Overview.md:73]: nit: Weird dot thing here as well.

File: docs/webhooks/10-V1-Overview.md [hunk 8]
```diff
@@ -637,5 +613,7 @@ title: incident.delegate
   ]
 }
 ```
+&nbsp;
+### Latest Version
 
-<!-- type: tab-end -->
+Please go to [V3 webhook](../webhooks/01-Overview.md)
```
Comment (ChezCrawford) [inline docs/webhooks/10-V1-Overview.md:619]: nit: Same thing about the wording

File: docs/webhooks/01-Overview.md [hunk 9]
```diff
@@ -455,3 +465,17 @@ Depending on the `event.event_type`, of the webhook payload, the `event.data` fi
   "type": "service"
 }
 ```
+&nbsp;
+## Deprecated Versions
+&nbsp;
+   ### V2 Reaching End Of Support
+   [V2 Webhooks](../webhooks/09-V2-Overview.md):
+   V2 will be reaching End-Of-Support(EOS) by October 2022.<br>
```
Comment (ChezCrawford) [inline docs/webhooks/01-Overview.md:473]: There is some weird spacing stuff here like `End-Of-Support(EOS)`.

File: docs/webhooks/09-V2-Overview.md (hunk 4 above)
Comment (nakul-bhagat) [reply docs/webhooks/09-V2-Overview.md:1142]: changed in the next PR.

File: docs/webhooks/09-V2-Overview.md (hunk 3 above)
Comment (nakul-bhagat) [reply docs/webhooks/09-V2-Overview.md:9]: changed in the next PR.

File: docs/webhooks/01-Overview.md (hunk 9 above)
Comment (nakul-bhagat) [reply docs/webhooks/01-Overview.md:473]: fixed in the next PR update.

File: docs/webhooks/01-Overview.md [hunk 10]
```diff
@@ -280,7 +290,7 @@ Sent when an incident is newly created/triggered.
 
 ### incident.unacknowledged
 
-`data.type` is [`incident`](#incident)
+`data.type` is [`incident`](##incident)
```
Comment (ChezCrawford) [inline docs/webhooks/01-Overview.md:293]: Is this correct? I think this breaks the link.

Comment (nakul-bhagat) [reply docs/webhooks/01-Overview.md:293]: typo.

Comment (nakul-bhagat) [reply docs/webhooks/01-Overview.md]: I removed it in the next PR.

**Review by ChezCrawford — APPROVED** [review]

**Pull request merged**